.circleci/config.yml

```diff
@@ -1513,66 +1151,57 @@ workflows:
               only:
                 - master
     jobs:
-      - node-core-12
-      - node-core-14
-      - node-core-16
-      - node-core-latest
+      - node-core: *matrix-supported-node-versions
```

Comment:
Can this list be extracted to a YAML anchor so that it doesn't need to be repeated between `build` and `nightly`?

Reply:
Basically like the benchmark jobs.

Reply:
Currently, the nightly doesn't include coverage reports. YAML anchors can't be used to extend arrays. If you're fine with just running coverage nightly, then we can re-use it. What do you think?

Reply:
I think for now it's fine if it makes this simpler. We can always reorganize later with more workflows if needed.

Reply:
done